beta-src/src/components/svgr-components/HomeIconSelected.js

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
```

Comment:
let's do the same thing we did to ActionIcon here and call this one just HomeIcon